dev/beaker/clone_gatling-puppet-load-test.rb

```diff
@@ -0,0 +1,9 @@
+test_name "Clone the gatling-puppet-load-test repo on the target machine"
+
+repo = "gatling-puppet-load-test"
+repo_url = "https://github.com/puppetlabs/#{repo}.git"
+
+hosts.each do |host|
```

Comment:
Technically we don't want to do this on all of the hosts, right?  I know that for now we only have one machine in the hosts file, so I'll probably be fine merging this as-is, just wondering if you have any thoughts on what this might look like if we did introduce multiple hosts (or if this beaker script were to end up getting used as part of the production environment, rather than just the dev env).